infra/modules/providers/azure/cosmosdb/tests/integration/cosmos.go

```diff
@@ -50,20 +47,20 @@ func getModuleOutputs(output infratests.TerraformOutput, outputName string) (id
 }
 
 // validateServiceResponse - Attempt to perform a HTTP request to the live endpoint.
-func validateServiceResponse(t *testing.T, output infratests.TerraformOutput, outputName string) {
-	_, endpoint, _, _ := getModuleOutputs(output, outputName)
-	statusCode, _ := httpClient.HttpGet(t, endpoint)
+// func validateServiceResponse(t *testing.T, output integration.TerraformOutput, outputName string) {
+// 	_, endpoint, _, _ := getModuleOutputs(output, outputName)
+// 	statusCode, _ := httpClient.HttpGet(t, endpoint)
```

Comment:
Looking at the code, I think adding a third argument of nil will do the trick. This argument refers to the TlsConfig. If nil, the default is used
 

https://github.com/gruntwork-io/terratest/blob/master/modules/http-helper/http_helper.go#L21

Is it possible to attempt to fix this? If it gives you trouble, I'm OK with commenting it for now.